Makefile

```diff
@@ -1,5 +1,9 @@
 DC ?= dmd
-DFLAGS += -w -g -ofonedrive -O -L-lcurl -L-lsqlite3 -L-ldl -J.
+ifdef NOTIFICATIONS
+	DFLAGSNOTIFICATIONS ?= -version=NoPragma -version=NoGdk -version=Notifications \
+		-L-lgmodule-2.0 -L-lglib-2.0 -L-lnotify
```

Comment:
What OS are these additional libraries the 'default' for? Debian?

Reply:
See above, I think this is the default for most distributions